src/olympia/amo/models.py

```diff
@@ -348,7 +348,8 @@ class ModelBase(SearchMixin, SaveUpdateMixin, models.Model):
     * Fetches all translations in one subsequent query during initialization.
     """
 
-    created = models.DateTimeField(auto_now_add=True)
+    created = models.DateTimeField(
+        default=timezone.now, editable=False, blank=True)
```

Comment:
I don't get that, this is what `auto_now_add` is doing anyway, https://docs.djangoproject.com/en/2.1/ref/models/fields/#django.db.models.DateField.auto_now_add or am I missing something?

Reply:
to quote from exactly that page:
> Note that the current date is always used; it’s not just a default value that you can override. So even if you set a value for this field when creating the object, it will be ignored. If you want to be able to modify this field, set the following instead of auto_now_add=True:
<what I've done>

Reply:
:man_facepalming: